src/movies.js

```diff
@@ -1,16 +1,113 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+function getAllDirectors(arr)
```

Comment:
Although this solution is correct, I need to point out that it doesn't follow any good practices with the syntax and code organization. We put certain things on a new line for a reason to make it visually better represented. This code could be turned into something like this:
```js
function getAllDirectors(movies) {
   const directors = movies.map(oneMovie => oneMovie.director);
   return directors;
}
```